plugins/qcheck-stm/test/dune

```diff
@@ -98,6 +105,26 @@
    (diff hashtbl_errors.expected hashtbl_errors)
    (diff hashtbl_stm_tests.expected.ml hashtbl_stm_tests.ml))))
 
+(rule
+ (target testing_warnings_errors)
+ (package ortac-qcheck-stm)
+ (deps
+  (package ortac-core)
```

Comment:
This should reuse the same set of dependencies and the `setenv` setting in the action than other tests.